game-backend/src/types/Pos.ts

```diff
@@ -0,0 +1,4 @@
+interface Pos {
```

Comment:
being nitpicky here: I usually prefer `Position` instead of the short version.

Reply:
The name Position got somehow merged with the default Typescript Position type. As I didn't know how to prevent that, I did it like this. Please tell me how I can resolve it.

Reply:
Updated it.